WebDriverAgentLib/Commands/FBSessionCommands.m

```diff
@@ -55,6 +55,8 @@ + (NSArray *)routes
     // Settings endpoints
     [[FBRoute GET:@"/appium/settings"] respondWithTarget:self action:@selector(handleGetSettings:)],
     [[FBRoute POST:@"/appium/settings"] respondWithTarget:self action:@selector(handleSetSettings:)],
+
+    [[FBRoute GET:@"/appium/device/info"] respondWithTarget:self action:@selector(handleGetDeviceInfo:)],
```

Comment:
```
[debug] [WD Proxy] Proxying [GET /appium/device/info] to [GET http://localhost:8100/session/3F3EEC09-FC0A-43C3-B120-38C87A37909B/appium/device/info] with no body
[debug] [WD Proxy] Got response with status 200: {
[debug] [WD Proxy]   "value" : {
[debug] [WD Proxy]     "currentLocale" : "en_US",
[debug] [WD Proxy]     "timeZone" : "Asia\/Tokyo"
[debug] [WD Proxy]   },
[debug] [WD Proxy]   "sessionId" : "3F3EEC09-FC0A-43C3-B120-38C87A37909B",
[debug] [WD Proxy]   "status" : 0
[debug] [WD Proxy] }
[debug] [W3C (a2877006)] Responding to client with driver.execute() result: {"currentLocale":"en_US","timeZone":"Asia/Tokyo"}
[HTTP] <-- POST /wd/hub/session/a2877006-58c3-4ad1-ba67-3045c8622343/execute/sync 200 14 ms - 59
[HTTP]
```

Reply:
is it expected that slash is escaped in `timeZone`?

Reply:
Yes. that was escaped on the log.
The output of client side is like below.
```
 {"currentLocale"=>"en_US", "timeZone"=>"Asia/Tokyo"}
```